tests/unit/test_model_snippets.py

```diff
@@ -123,14 +126,17 @@ def test_get_deployment_snippet_kubernetes(mock_client):
         {
             "repoName": "meta-llama/Llama-4-Maverick-17B-128E-Instruct",
             "configsDeploy": {
-                "xe9680-amd-mi300x": [
-                    {
-                        "max_batch_prefill_tokens": 16484,
-                        "max_input_tokens": 16383,
-                        "max_total_tokens": 16384,
-                        "num_gpus": 8,
-                    }
-                ]
+                "containerTags": {},
```

Comment:
Nit: we could add actual values for `containerTags` for the tests, see below

P.S. The suggestion below won't work, is just for reference!

```suggestion
                "containerTags": {
                    "nvidia": [
				        {"id": "latest", "containsWeights": True},
				        {"id": "tgi-3.0.2", "containsWeights": False},
				    ],
				    ...
                },
```